src/libstd/unstable/dynamic_lib.rs

```diff
@@ -45,12 +45,22 @@ impl Drop for DynamicLibrary {
 }
 
 impl DynamicLibrary {
+    // FIXME (#12938): Until DST lands, we cannot decompose &str into
```

Comment:
@aturon this comment is still in the codebase, even though ToCStr is gone and `to_os_str` is an inherent method on `Path`. Should this comment just be removed?